examples/README.md

```diff
@@ -9,6 +10,11 @@ for llama2
 ```bash
 CUDA_VISIBLE_DEVICES=0,1 python -m torch.distributed.run --nproc_per_node 2 examples/rap_gsm8k/inference.py --llama_size "13B" --output_trace_in_each_iter --base_lm 'llama2'
 ```
+for exllama and huggingface llama(params need to fill in inference.py)
```

Comment:
why are the params set in the py file?

Reply:
It's just because I want users to check which features they'd like to turn on.